plugins/poll/app/models/poll.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class Poll < ActiveRecord::Base
+  # because we want to use the 'type' column and don't want to use STI
+  self.inheritance_column = nil
+
+  belongs_to :post
+
+  has_many :poll_options, dependent: :destroy
+  has_many :poll_votes, dependent: :delete_all
```

Comment:
Maybe we don't need this? `poll_options` owns the `poll_votes` and has a `dependent: :delete_all`

Reply:
This allows me to count the # of vote(r)s directly on the `Poll` object instead of going through all the `PollOption`.